code/framework/api/src/main/java/io/cattle/platform/api/servlet/ApiRequestFilter.java

```diff
@@ -37,6 +37,9 @@
     private static final DynamicStringListProperty IGNORE = ArchaiusUtil.getList("api.ignore.paths");
     private static final DynamicStringProperty PL_SETTING = ArchaiusUtil.getString("ui.pl");
     private static final String PL = "PL";
+    private static final String LOCALIZATION = System.getenv("CATTLE_LOCALIZATION");
```

Comment:
This should be
```java
      private static final DynamicStringProperty LOCALIZATION = ArchaiusUtil.getString("localization");
```

What this means is that the env variable `CATTLE_LOCALIZATION` will set it, or you can set `DEFAULT_CATTLE_LOCALIZATION` to set the default value but still let the user override it.  Typically we set `DEFAULT_CATTLE_*` in the dockerfile to prepopulate settings.